src/ios-plugin/pom.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.ecocode</groupId>
+    <artifactId>ecocode-ios-rules</artifactId>
+    <version>0.1-SNAPSHOT</version>
```

Comment:
Ok, I remove it for now. But that would mean that every plugin (android and ios) in ecoCode-mobile must evolve in sync ? (every time Android v x.y.z is released iOS v x.y as well) 

Anyway, I think we should separate android and ios plugins in 2 separated repos otherwise we break almost any packaging architecture principles (REP, CRP, CCP) because:
- we don't necessary want to release same versions on android / ios at the same time
- we may have to release fixes on only one of the two plugins: in this case a single version number does not work
- there is not a single line of code in common between the 2 plugins

What to you think @dedece35  ?

Reply:
I agree with you and not agree at the same time :p 
We discussed about this architecture principle a couple of weeks ago during a friday session with core team members.
Maybe we can discuss about mobile part (and standard part also) again ... to expose your point of view.
You can ask questions on "project-architecture" channel on Slack.

Reply:
you can also participate to our weekly core team meeting to discuss about it.